dipy/core/tests/test_sphere.py

```diff
@@ -357,5 +358,28 @@ def test_disperse_charges():
     nt.assert_array_almost_equal(norms, 1)
 
 
+def test_disperse_charges_alt():
+    # Create a random set of points
+    charges = np.random.random((3, 3))
```

Comment:
Here you may want to use `dipy.core.sphere_stats.random_uniform_on_sphere`, which does sample from a uniform distribution on the sphere. 

What about having a variable `nb_points`?

Reply:
:thumbsup: Thanks @ecaruyer ! Makes the test easier to read. changed in ebf9d83.